### DISCUSSION TITLE
https://xstate.js.org/viz/ open-sourced?

### DISCUSSION THREAD

**Discussion by xiaoyu2er** [statelyai/xstate#1396] [General] [answered]
https://xstate.js.org/viz/ is verify useful
is this website open-sourced @davidkpiano

**Comment by Andarist** [comment, marked as answer]
It's here https://github.com/statecharts/xstate-viz . Keep in mind that the visualizer is currently rebuilt by @davidkpiano

**Reply by davidkpiano** [reply]
Yep - the new visualizer will be in this repo, under `packages/xstate-viz`.

**Reply by xiaoyu2er** [reply]
statecharts/xstate-viz is not latatest
hope packages/xstate-viz publish soon
